tests/ssl-opt.sh

```diff
@@ -308,6 +308,7 @@ if type lsof >/dev/null 2>/dev/null; then
         done
     }
 else
+    echo "Warning: lsof not available, wait_server_start = sleep 1"
```

Comment:
Please change this to `… = sleep $START_DELAY`

Reply:
Haha, that's funny, I remember thinking that when looking at your PR, but then forgot about it when cherry-picking. Thanks for noticing, I'll fix that.